custom_components/ithodaalderop/config_flow.py

```diff
@@ -89,7 +86,7 @@ async def async_step_rooms(self, info=None):
         """Configure rooms for autotemp."""
         if info is not None:
             self.config.update(info)
-            await self.async_set_unique_id("ithoaddon")
+            await self.async_set_unique_id(self.config[CONF_ADDON_TYPE])
```

Comment:
This might clash with `await self.async_set_unique_id(self.config[CONF_ADDON_TYPE])` for the unit itself